docs/modules/ROOT/pages/explanations/eventforwarding.adoc

```diff
@@ -0,0 +1,22 @@
+= Event Forwarding
+
+== How does it work?
+
+We use a controller to catch all events on a cluster.
+We check the owner object of every event for a specific annotation.
+
+The annotation contains the kind, namespace and name of a claim that should receive the event.
+
+
+== Annotation
+
+The annotation is called `vshn.net/forward-events-to` and has the following format: `<claim_kind>/<claim_namespace>/<claim_name>`.
```

Comment:
I would add `appcat` word in here. We need to make sure this works only with appcat.

Reply:
changed the annotation to `appcat.vshn.io/forward-events-to`